src/popup.html

```diff
@@ -58,12 +50,13 @@ <h3><a href="https://www.mhct.win" target="_blank" class="popup-link-out">MH Com
                 Options
             </td></tr>
             <tr><td>
-                <a href="popup2.html" class="popup-link-out"><div class="pull-right">&gt;&gt;</div>Other Links</a>
```

Comment:
isn't html encoded is usually more compatible than using special characters directly?

Reply:
`>>` isn't correct for what this is though. Unicode characters have full support every where unless they're in an extended set, which isn't the case for this arrow character.